src/test/java/catalinaTema5/DemoqaElements/Buttons.java

```diff
@@ -0,0 +1,24 @@
+package catalinaTema5.DemoqaElements;
+
+import catalinaTema5.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Buttons {
+    public static void main(String[] args) {
+        goToButtonsPage();
+   }
+
+    public static void goToButtonsPage(){
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/buttons");
+        WebElement doubleClickMe = driver.findElement(By.id("doubleClickBtn"));
+        doubleClickMe.click();
+        WebElement rightClickMe = driver.findElement(By.id("rightClickBtn"));
+        rightClickMe.click();
+        WebElement clickMe = driver.findElement(By.className("btn btn-primary));
```

Comment:
Use By.cssSelector(".btn.btn-primary") as By.className won't work for compounded class names.